source/codegen/metadata/nidmm/functions.py

```diff
@@ -1,4 +1,20 @@
 functions = {
+    "Control4022":{
```

Comment:
@jonlugoatx @reckenro Using this [API](https://zone.ni.com/reference/en-XX/help/370384V-01/dmmviref/nidmm_4022_control/) as is (4022Control) gives errors in the proto file. Hence I have used it this way and added a cname tag.

Reply:
That seems fine to me. You should reach out to a driver expert in DOE to be sure, but that feedback shouldn't gate this review.